### PR TITLE
jellyfin: 10.5.3 -> 10.5.4, add link to nixos test

### PR DESCRIPTION
**Pull request by minijackson**
###### Motivation for this change

I think this is how you link nixos tests ^^

cc @nyanloutre 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by nyanloutre** [comment]
I tested this on my system, it's working great

File: pkgs/servers/jellyfin/default.nix [hunk 1]
```diff
@@ -55,4 +55,8 @@ in stdenv.mkDerivation rec {
     license = licenses.gpl2;
     maintainers = with maintainers; [ nyanloutre minijackson ];
   };
+
+  passthru.tests = {
+    inherit (nixosTests) jellyfin;
+  };
```
Comment (worldofpeace) [inline pkgs/servers/jellyfin/default.nix:61]: can we order the passthru attribute above `meta`? Just an aesthetic preference.
If we `nix-build` this it looks a little funny
```
nix-build -A jellyfin.tests.jellyfin
```

could you do something like
```
{
  what-this-test-does = jellyfin;
};
```

just for a bit more information about the test case?

**Comment by worldofpeace** [comment]
This is a hotfix release https://github.com/jellyfin/jellyfin/releases/tag/v10.5.4.
Could we also backport it to the stable channel?

**Comment by minijackson** [comment]
> Could we also backport it to the stable channel?

I'm all for it! Do I need to do something for the backport?

**Comment by worldofpeace** [comment]
> > Could we also backport it to the stable channel?
> 
> I'm all for it! Do I need to do something for the backport?

See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#backporting-changes

**Comment by minijackson** [comment]
So I just upgraded my server to use jellyfin 10.5.4, and it failed to start the service because of the "CinemaMode" plugin that I carried all the way from the migration from Emby. A simple solution is to move / remove the file `/var/lib/jellyfin/Emby.Server.CinemaMode.dll`. I don't know if we should do something packaging-wise, since these kinds of plugins come from Emby, and are [not supported](https://github.com/jellyfin/jellyfin/issues/480).

I could make a PR on the [migration guide](https://jellyfin.org/docs/general/administration/migrate-from-emby.html) to hint people that they should remove these plugins.

**Comment by worldofpeace** [comment]
> So I just upgraded my server to use jellyfin 10.5.4, and it failed to start the service because of the "CinemaMode" plugin that I carried all the way from the migration from Emby. A simple solution is to move / remove the file `/var/lib/jellyfin/Emby.Server.CinemaMode.dll`. I don't know if we should do something packaging-wise, since these kinds of plugins come from Emby, and are [not supported](https://github.com/jellyfin/jellyfin/issues/480).
> 
> I could make a PR on the [migration guide](https://jellyfin.org/docs/general/administration/migrate-from-emby.html) to hint people that they should remove these plugins.

I once upon a time had to workaround a permissions issue in emby https://github.com/NixOS/nixpkgs/commit/a16c6d85a1291beede624cdc14833247610eb706. so handling things in that way can sometimes be easy.
Given that migration from emby isn't officially supported, mentioning this in that page would be a good idea.

**Pull request merged**